src/app/blog/healing-connections/page.mdx

```diff
@@ -0,0 +1,99 @@
+import { BlogPostLayout } from '@/components/BlogPostLayout'
+import { MotionCanvas } from '@/components/MotionCanvas'
+
+export const post = {
+  draft: false,
+  author: 'ramfox',
+  date: '2024-06-17',
+  title: 'Healing Connections After Network Migration',
+  description: 'Describes the process that iroh-net uses to heal a connection after one side has changed networks or IP addresses.',
+}
+
+export const metadata = {
+    title: post.title,
+    description: post.description,
+    openGraph: {
+      title: post.title,
+      description: post.description,
+      images: [{
+        url: `/api/og?title=Blog&subtitle=${post.title}`,
+        width: 1200,
+        height: 630,
+        alt: post.title,
+        type: 'image/png',
+      }],
+      type: 'article'
+    }
+}
+
+export default (props) => <BlogPostLayout article={post} {...props} />
+
+Iroh is an open-source distributed systems toolkit with tools for connecting devices directly, moving data, and syncing state.
+
+Iroh’s networking stack can do a lot of clever things. It can allow you to dial by NodeID, using [certain discovery mechanisms]([https://iroh.computer/blog/iroh-dns](https://iroh.computer/blog/iroh-dns)). It allows you to communicate with another machine, even if your computer and the other computer are each behind a NAT, using hole punching. 
```

Comment:
```suggestion
Iroh’s networking stack can do a lot of clever things. It can allow you to dial by NodeID, using [certain discovery mechanisms](https://iroh.computer/blog/iroh-dns). It allows you to communicate with another machine, even if your computer and the other computer are each behind a NAT, using hole punching. 
```